DVMultiplayerContinued/DVMultiplayerContinued.csproj

```diff
@@ -166,7 +166,6 @@
     <Compile Include="Darkrift\Client\UnityClient.cs" />
     <Compile Include="Darkrift\Server\UnityModManagerWriter.cs" />
     <Compile Include="Darkrift\Server\UnityConsoleWriter.cs" />
-    <Compile Include="Darkrift\Server\UnityServer.cs" />
```

Comment:
Removed in commit e173dc17b7b43f3669fdc7b9f0c29a2b44c8cf87 but not unreferenced here. Build fails in Visual Studio.